src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -405,48 +461,69 @@
   [community-content message])
 
 (defmethod ->message constants/content-type-status
-  [{:keys [content content-type] :as message}]
+  [{:keys [content content-type pinned] :as message}]
   [message-content-wrapper message
    [react/view style/status-container
     [react/text {:style (style/status-text)}
      (reduce
-      (fn [acc e] (render-inline (:text content) false content-type acc e))
+      (fn [acc e] (render-inline (:text content) false pinned content-type acc e))
       [react/text-class {:style (style/status-text)}]
       (-> content :parsed-text peek :children))]]])
 
 (defmethod ->message constants/content-type-emoji
-  [{:keys [content current-public-key outgoing public?] :as message} {:keys [on-long-press modal]
-                                                                      :as   reaction-picker}]
+  [{:keys [content current-public-key outgoing public? pinned in-popover?] :as message} {:keys [on-long-press modal]
+                                                                                         :as   reaction-picker}]
   (let [response-to (:response-to content)]
     [message-content-wrapper message
      [react/touchable-highlight (when-not modal
-                                  {:on-press      (fn []
+                                  {:disabled      in-popover?
+                                   :on-press      (fn []
                                                     (react/dismiss-keyboard!))
                                    :on-long-press (fn []
-                                                    (on-long-press
-                                                     [{:on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
-                                                       :label    (i18n/label :t/message-reply)}
-                                                      {:on-press #(react/copy-to-clipboard (get content :text))
-                                                       :label    (i18n/label :t/sharing-copy-to-clipboard)}]))})
+                                                    (let [{:keys [chat-id group-chat community-id admins]} @(re-frame/subscribe [:chats/current-chat-chat-view])
```

Comment:
it seems like these all are chat related, not message related, so we must subscribe only once in chat view and pass this data to every message

Reply:
Done, added as additional data on each message injected on flat list